keyboards/h0oni/deskpad/readme.md

```diff
@@ -0,0 +1,12 @@
+# Hydrogen Deskpad
+
+A 5 key macropad with an encoder, that can be mounted to a desk or below your monitor.
+
+* Keyboard Maintainer: Hydrogen BD
```

Comment:
No links?

Reply:
Website is not up yet. I have facebook and insta links only. Should I add those? what do you recommend.